### PR TITLE
Shashi assignment files

### PR DESCRIPTION
**Pull request by srtirumala**
Adding my assignment files for :

• arrayMethods
• loops
• shop assignment

### CODE REVIEW

File: shashi/arrayMethods.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+var inputArray = [1, 2, 3, 4, 5, 6];
+
+function indexOf(inputArray, value) {
```
Comment (koding-101) [inline shashi/arrayMethods.js]: What happens if the value is not in the array?

File: shashi/arrayMethods.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      return i;
+    }
+  }
+}
+
+console.log(indexOf(inputArray, 5));
+
+var str = 'my%name%is%Shashi%Reddy%Tirumala%23';
+
+function split(string, seperator) {
```
Comment (koding-101) [inline shashi/arrayMethods.js:53]: Try to do this with only one loop

File: shashi/loops.js [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+  return backWordArr;
+}
+
+function backwordsWhile(array){
+  var backWordArr = [];
+  var i = array.length-1;
+  while(i>=0){
+    backWordArr.push(array[i]);
+    i--;
+    }
```
Comment (koding-101) [inline shashi/loops.js:69]: indent

File: shashi/loops.js [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+backwordsWhile(arr);
+
+var arr1 = [6,7,8,9,10];
+var arr = [1,2,3,4,5];
+
+//combine both arrays
+
+function combineArr(array1,array2){
+  var combinedArr =[];
+   combinedArr= array1;
```
Comment (koding-101) [inline shashi/loops.js]: This does not create a new array for combinedArr. In console, try this and push to combinedArr and then see what the value of array1 is.

File: shashi/loops.js [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+function alternate(array1, array2){
+  var maxLength = 0;
+  var altArr=[];
+  if(array1.length >= array2.length){
+    maxLength = array1.length;
+  } else {
+    maxLength = array2.length;
+  }
+  for (var i = 0; i < maxLength; i++) {
+    altArr.push(array1[i], array2[i]);
```
Comment (koding-101) [inline shashi/loops.js]: What would happen here if one of the array is shorter than the other?

File: shashi/loops.js [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+    var altArr=[];
+    if(array1.length >= array2.length){
+      maxLength = array1.length;
+    } else {
+      maxLength = array2.length;
+    }
+    console.log(maxLength);
+    var i=0;
+    while(i < maxLength){
+      altArr.push(array1[i], array2[i]);
```
Comment (koding-101) [inline shashi/loops.js]: Same here

**Comment by srtirumala** [comment]
@koding-101 Pushed changes as per review comments and added HTML page. Please review :)

**Pull request merged**